src/cljs/main/broadfcui/page/workspace/summary/catalog/questions.cljs

```diff
@@ -109,49 +110,70 @@
                            :rows 3
                            :data-test-id property})) ;; Dataset attribute, looks like "library:datasetOwner"
 
-(defn- render-ontology-typeahead [{:keys [prop colorize value-nullsafe set-property state property library-schema disabled]}]
-  (let [clear #(apply swap! state update :attributes dissoc property
-                      (library-utils/get-related-id+label-props library-schema property))]
+
+(def ^:private ATTRIBUTE_SEPARATOR ", ")
+
+(defn- split-attributes [values]
+  (string/split values (re-pattern ATTRIBUTE_SEPARATOR)))
+
+(defn- zip-comma-separated-strings [keys values]
+  (if (string/blank? keys)
+    {}
+    (let [split-keys (split-attributes keys)
+          split-values (split-attributes values)]
+        (zipmap split-keys split-values))))
+
+(defn- remove-from-comma-separated-strings [value-string item]
+  (if (string/blank? value-string)
+    value-string
+    (let [values (split-attributes value-string)]
+      (string/join ATTRIBUTE_SEPARATOR (utils/delete values (utils/index-of values item))))))
+
+
+(defn- render-ontology-typeahead [{:keys [refs prop colorize value-nullsafe set-property state property library-schema disabled]}]
+  (let [[related-id-prop related-label-prop] (library-utils/get-related-id+label-props library-schema property)
```

Comment:
inside `get-related-id+label-props` would be a good place to ensure both attrs have the same length, and take action if they don't.